src/plugins/discover/public/application/application.ts

```diff
@@ -12,28 +12,9 @@ import { renderApp as renderReactApp } from './index';
  * Here's where Discover is mounted and rendered
  */
 export async function renderApp(moduleName: string, element: HTMLElement) {
```

Comment:
This method does not do much anymore. Could we instead just get rid of the whole file and import `renderApp` from the `index.tsx` file in this same folder directly? (It seems to be only one place we're importing it `plugin.tsx`).